src/guides/resource-records.md

```diff
@@ -0,0 +1,265 @@
+## Handshake Resource Record Guide
+
+Handshake has resource records that represent DNS resource records.
+Some of them map one-to-one with DNS resource record types,
+while others do not. These resource records are included in
+`REGISTER` or `UPDATE` outputs and written to the Urkel tree
+so that proofs can be served to light clients.
+
+Each Handshake name stores a single `Resource` with the maximum size of
+512 bytes.
+
+### Updating Resource Records
+
+To update the resource records associated with a name, the wallet RPC
+command `sendupdate` is used. If the name is in a different wallet
+than the primary wallet, use the RPC command `selectwallet` before
+sending the update. The Node RPC method `verifyresource` can be used
+to check if the resource is valid before attempting to `sendupdate`.
+
+```bash
+$ hsw-cli rpc sendupdate easyhandshake '{"records": []}'
+```
```

Comment:
Explain that this is an empty record and can be used to get your change from the second-highest bid without adding records (maybe you arent ready yet, or you're just holding the name for resale instead of use)

The thing with examples is, users will copy+paste the first example they see, and then ask on telegram why their name doesn't resolve :-)

Reply:
I added a blurb that explains the gotchas around this, including noting that this includes no actual records.